engine/engine.go

```diff
@@ -40,6 +41,12 @@ func NewEnv(opts docker.Opts) (*Engine, error) {
 }
 
 func (e *Engine) Setup(ctx context.Context, pipelineConfig *spec.PipelineConfig) error {
+	// create global files and folders
```

Comment:
Create files in handler/setup.go.

Reply:
@shubham149 I disagree. Handler code should deal with HTTP stuff: Parsing request body and parameters, and not the business logic. Architecturally it's better to place this code here - in the engine, because it's manipulating the VM instance.